about.html

```diff
@@ -1,7 +1,10 @@
+These are our house rules. Please read them before signing up. Any violation of any of those may lead to deletion of toots, silencing or suspension of accounts, at the descretion of the instance administrators. Usually, we'd tell you a couple of times in advance, before taking any of these measures.
```

Comment:
It was a very conscious decision to start our about page with "Be excellent to each other". This paragraph should e moved up to be the introduction instead of this one.

Reply:
Yes it makes sense. I also like to have this sentence first. I tried to rephrase it to have it first but still make the enforcement paragraph understandably applikable to all rules (including the automated posts).